capture/socketClient.go

```diff
@@ -2,14 +2,16 @@ package capture
 
```

Comment:
the only changes I made to this file are formatting and linting related, namely to shut my IDE up lmao.